backup-tools/backup-restore/app.go

```diff
@@ -71,12 +76,12 @@ func startRestoreInit(cc bool) error {
 		bufio.NewReader(os.Stdin).ReadBytes('\n')
 	}
 	if os.Getenv("BACKUP_PGSQL_FULL") != "" {
-		backupType = "pgsql"
+		utils.BackupType = "pgsql"
```

Comment:
There is no support for the etcd backup. Because you normally start a cluster with the backup volume for a test startup and then restart with the restored volume as usual.